platform/platform-filter-clientinfo/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+    <bean id="clientInfoFilter"
+          class="org.codice.ddf.platform.filter.clientinfo.impl.ClientInfoFilter"/>
+    <service ref="clientInfoFilter" interface="javax.servlet.Filter">
```

Comment:
Providing this service with a `ranking=10` brings us closer to correct functionality, but only for create requests.